politeiawww/api/v1/api.md

```diff
@@ -1420,6 +1423,7 @@ Reply:
 | <a name="ErrorStatusInvalidInput">ErrorStatusInvalidInput</a> | 24 | Invalid input. |
 | <a name="ErrorStatusInvalidSigningKey">ErrorStatusInvalidSigningKey</a> | 25 | Invalid signing key. |
 | <a name="ErrorStatusCommentLengthExceededPolicy">ErrorStatusCommentLengthExceededPolicy</a> | 26 | The submitted comment length is too large. |
+| <a name="ErrorStatusUserNotPaid">ErrorStatusUserNotPaid</a> | 28 | User not paid paywall. |
```

Comment:
It looks like the value is `30` [in this code](https://github.com/decred/politeia/pull/229/files#diff-4453ae1d7a0099c6b34bbd88b3ec783cR120), not `28`? Would you mind also adding the 2 missing errors to the docs?